template/src/main/java/com/asyncapi/model/$$objectSchema$$.java

```diff
@@ -146,8 +146,8 @@ public class {{allName}} {
     {%- if propName | isRequired(schema.required()) %}@NotNull{% endif %}
     {%- if prop.minLength() or prop.maxLength() or prop.maxItems() or prop.minItems() %}@Size({% if prop.minLength() or prop.minItems() %}min = {{prop.minLength()}}{{prop.minItems()}}{% endif %}{% if prop.maxLength() or prop.maxItems() %}{% if prop.minLength() or prop.minItems() %},{% endif %}max = {{prop.maxLength()}}{{prop.maxItems()}}{% endif %}){% endif %}    
     {%- if prop.pattern() %}@Pattern(regexp="{{prop.pattern() | addBackSlashToPattern}}"){% endif %}
-    {%- if prop.minimum() %}@Min({{prop.minimum()}}){% endif %}{% if prop.exclusiveMinimum() %}@Min({{prop.exclusiveMinimum() + 1}}){% endif %}
-    {%- if prop.maximum() %}@Max({{prop.maximum()}}){% endif %}{% if prop.exclusiveMaximum() %}@Max({{prop.exclusiveMaximum() + 1}}){% endif %}
+    {%- if prop.minimum() %}@Min({{prop.minimum()}}){% endif %}{% if prop.exclusiveMinimum() %}@DecimalMin({{prop.exclusiveMinimum() + 1}}){% endif %}
+    {%- if prop.maximum() %}@Max({{prop.maximum()}}){% endif %}{% if prop.exclusiveMaximum() %}@DecimalMax({{prop.exclusiveMaximum() + 1}}){% endif %}
```

Comment:
The `prop.minimum()` means `<=`, the `prop.maximum()` means `>=` and `exclusiveMinimum` means just `<` and `exclusiveMaximum` is `>`.
So, change of exclusiveMinimum and exclusiveMaximum to `@DecimalMin`/`@DecimalMax` will lead to effect that all types (Integer, Long, Double e.t.c.) will have the `@DecimalMin` annotation.
The correct way to choose annotation between `@Min` and `@DecimalMin`, `@Max` and `@DecimalMax` would require to analyze value of {{propType}} in line below. Depending on {{propType}} the correct annotation should be set